terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -13,13 +13,298 @@
     depth_bottom = models.IntegerField(blank=True)
+
+    SAND = "sand"
+    LOAMY_SAND = "loamy sand"
+    SANDY_LOAM = "sandy loam"
+    SILT_LOAM = "silt loam"
+    SILT = "silt"
+    LOAM = "loam"
+    SANDY_CLAY_LOAM = "sandy clay loam"
+    SILTY_CLAY_LOAM = "silty clay loam"
+    CLAY_LOAM = "clay loam"
+    SANDY_CLAY = "sandy clay"
+    SILTY_CLAY = "silty clay"
+    CLAY = "clay"
+
+    TEXTURES = (
+        (SAND, _("Sand")),
+        (LOAMY_SAND, _("Loamy sand")),
+        (SANDY_LOAM, _("Sandy loam")),
+        (SILT_LOAM, _("Silt loam")),
+        (SILT, _("Silt")),
+        (LOAM, _("Loam")),
+        (SANDY_CLAY_LOAM, _("Sandy clay loam")),
+        (SILTY_CLAY_LOAM, _("Silty clay loam")),
+        (CLAY_LOAM, _("Clay loam")),
+        (SANDY_CLAY, _("Sandy clay")),
+        (SILTY_CLAY, _("Silty clay")),
+        (CLAY, _("Clay")),
+    )
+    texture = models.CharField(null=True, choices=TEXTURES)
+
+    ROCK_FRAGMENT_0_1 = "0 — 1%"
+    ROCK_FRAGMENT_1_15 = "1 — 15%"
+    ROCK_FRAGMENT_15_35 = "15 — 35%"
+    ROCK_FRAGMENT_35_60 = "35 — 60%"
+    ROCK_FRAGMENT_60 = "> 60%"
+
+    ROCK_FRAGMENT_VOLUMES = (
+        (ROCK_FRAGMENT_0_1, _("0 — 1%")),
+        (ROCK_FRAGMENT_1_15, _("1 — 15%")),
+        (ROCK_FRAGMENT_15_35, _("15 — 35%")),
+        (ROCK_FRAGMENT_35_60, _("35 — 60%")),
+        (ROCK_FRAGMENT_60, _("> 60%")),
+    )
+    rock_fragment_volume = models.CharField(null=True, choices=ROCK_FRAGMENT_VOLUMES)
+
+    SUBSTEP_2_5 = "2.5"
+    SUBSTEP_5 = "5"
+    SUBSTEP_7_5 = "7.5"
+    SUBSTEP_10 = "10"
+
+    COLOR_HUES_SUBSTEPS = (
+        (SUBSTEP_2_5, _("2.5")),
+        (SUBSTEP_2_5, _("5")),
+        (SUBSTEP_7_5, _("7.5")),
+        (SUBSTEP_10, _("10")),
+    )
+    color_hue_substeps = models.CharField(null=True, choices=COLOR_HUES_SUBSTEPS)
+
+    RED = "R"
+    YELLOW_RED = "YR"
+    YELLOW = "Y"
+    GREEN_YELLOW = "GY"
+    GREEN = "G"
+    BLUE = "B"
+    BLUE_GREEN = "BG"
+
+    COLOR_HUES = (
+        (RED, _("r")),
+        (YELLOW_RED, _("yr")),
+        (YELLOW, _("y")),
+        (GREEN_YELLOW, _("gy")),
+        (GREEN, _("g")),
+        (BLUE, _("b")),
+        (BLUE_GREEN, _("bg")),
+    )
+    color_hue = models.CharField(null=True, choices=COLOR_HUES)
+
+    COLOR_VALUE_2_5 = "2.5"
+    COLOR_VALUE_3 = "3"
+    COLOR_VALUE_4 = "4"
+    COLOR_VALUE_5 = "5"
+    COLOR_VALUE_6 = "6"
+    COLOR_VALUE_7 = "7"
+    COLOR_VALUE_8 = "8"
+    COLOR_VALUE_8_5 = "8.5"
+    COLOR_VALUE_9 = "9"
+    COLOR_VALUE_9_5 = "9.5"
+
+    COLOR_VALUES = (
+        (COLOR_VALUE_2_5, _("2.5")),
+        (COLOR_VALUE_3, _("3")),
+        (COLOR_VALUE_4, _("4")),
+        (COLOR_VALUE_5, _("5")),
+        (COLOR_VALUE_6, _("6")),
+        (COLOR_VALUE_7, _("7")),
+        (COLOR_VALUE_8, _("8")),
+        (COLOR_VALUE_8_5, _("8.5")),
+        (COLOR_VALUE_9, _("9")),
+        (COLOR_VALUE_9_5, _("9.5")),
+    )
+    color_value = models.CharField(null=True, choices=COLOR_VALUES)
```

Comment:
Let's store this as a decimal field so we can do aggregate operations (sum, average, etc.).